src/App.tsx

```diff
@@ -130,12 +152,16 @@ function App() {
               id="difficulty-setting"
               type="range"
               min="0"
-              max="2"
+              max="4"
               value={difficulty}
               onChange={(e) => setDifficulty(+e.target.value)}
             />
             <div>
               <label htmlFor="difficulty-setting">Difficulty:</label>
+              &nbsp;
+              <strong>
+                {["Baby", "Easy", "Normal", "Hard", "Ultra Hard"][difficulty]}
+              </strong>
               <strong>{["Normal", "Hard", "Ultra Hard"][difficulty]}</strong>
```

Comment:
I haven't tried this, but I think this line with the 3 old difficulty values is probably a mistake. As the line above has the new 5. So I guess this line will show them twice in the modes > Easy.

By the way, I guess the `&nbsp;` 4 lines above was not needed before, and I guess is still not needed. Or does this solve a separate issue?